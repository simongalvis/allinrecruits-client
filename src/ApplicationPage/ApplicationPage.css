.ApplicationPage{
    height: 100vh;
}