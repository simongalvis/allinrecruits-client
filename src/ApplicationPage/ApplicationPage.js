import React from 'react';
import NavBar from '../NavBar/NavBar';
import './ApplicationPage.css'


class ApplicationPage extends React.Component{

    render(){
        return(
            <div className="ApplicationPage">
                <NavBar/>
                <h1>Application Page</h1>
                <h2>React type effect hero here</h2>
            </div>
        )
    }
}

export default ApplicationPage;