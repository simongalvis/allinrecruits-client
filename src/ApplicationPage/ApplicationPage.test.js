import React from 'react'
import ReactDOM from 'react-dom'
import ApplicationPage from './ApplicationPage'
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom'

describe(`ApplicationPage component`, () =>{
    it(`renders without crashing`, () =>{
        const div = document.createElement('div');

        ReactDOM.render(<BrowserRouter><ApplicationPage/></BrowserRouter>, div)

        ReactDOM.unmountComponentAtNode(div);
    })
    it(`Renders UI as expected`, () =>{
        const tree = renderer
            .create(<BrowserRouter><ApplicationPage/></BrowserRouter>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})