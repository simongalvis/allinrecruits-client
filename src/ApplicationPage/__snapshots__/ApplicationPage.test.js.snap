// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationPage component Renders UI as expected 1`] = `
<div
  className="ApplicationPage"
>
  <div
    className="NavBar"
  >
    <header>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="all-in-education-logo"
          id="main-logo"
          src="all-in-logo.png"
        />
      </a>
      <a
        href="/admin-login"
      >
        <button
          id="admin-login-btn"
        >
          Admin Login
        </button>
      </a>
    </header>
  </div>
  <h1>
    Application
  </h1>
  <div
    className="ApplicationForm"
  >
    <form
      onSubmit={[Function]}
    >
      <label
        className="required"
        htmlFor="applicationFullname"
      >
        Full name:
      </label>
      <br />
      <input
        autoCapitalize="none"
        autoCorrect="off"
        id="applicationFullname"
        name="applicationFullname"
        placeholder="John Smith"
        required={true}
        type="text"
      />
      <br />
      <label
        htmlFor="applicationPhonenumber"
      >
        Phone number:
      </label>
      <br />
      <input
        autoCapitalize="none"
        autoCorrect="off"
        id="applicationPhonenumber"
        name="applicationPhonenumber"
        placeholder="123-456-7890"
        type="tel"
      />
      <br />
      <label
        htmlFor="application-email"
      >
        Email:
      </label>
      <br />
      <input
        autoCapitalize="none"
        autoCorrect="off"
        id="applicationEmail"
        name="applicationEmail"
        placeholder="johnsmith123@email.com"
        type="email"
      />
      <br />
      <label
        htmlFor="applicationPosition"
      >
        Interested Position:
      </label>
      <br />
      <select
        id="applicationPosition"
        name="applicationPosition"
      >
        <option
          value="Spanish teacher"
        >
          Spanish teacher
        </option>
        <option
          value="French teacher"
        >
          French teacher
        </option>
        <option
          value="Robotics instructor"
        >
          Robotics instructor
        </option>
        <option
          value="Art instructor"
        >
          Art instructor
        </option>
      </select>
      <br />
      <label
        className="required"
        htmlFor="applicationResume"
      >
        Resume(Link)
      </label>
      <br />
      <input
        autoCapitalize="none"
        autoCorrect="off"
        id="applicationResume"
        name="applicationResume"
        placeholder="https://drive.google.com/file/d/examplelink3648384748/view"
        required={true}
        type="url"
      />
      <br />
      <button
        id="application-submit-btn"
        type="submit"
      >
        <b>
          Apply
        </b>
      </button>
    </form>
  </div>
</div>
`;
